### PR TITLE
Quiz question

### PR DESCRIPTION
**Pull request by Ankur9669**
Files to check:
1. QuizQuestion.jsx
2.  quizzes.js

image

### CODE REVIEW

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+const QuizQuestion = () => {
+  const { quiz, setQuiz, answersSelected, setAnswersSelected } = useQuiz();
+  const navigate = useNavigate();
+  const { showToast } = useToast();
+
+  //   if (quiz === null) {
+  //     navigate("/");
+  //     return null;
+  //   }
```
Comment (Naman-Saxena1) [inline quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: If not using, can remove it

**Review by Naman-Saxena1 — APPROVED** [review]
This is small PR, code and UI looks good 👍

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 1 above)
Comment (Ankur9669) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: Done

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+  const quizName = quiz?.quizName;
+  const quizImage = quiz?.quizImage;
+  const [questionToShow, setQuestionToShow] = useState(0);
+  const questions = quiz?.questions;
+  const question = questions?.[questionToShow];
+  const options = question?.options;
+
+  // console.log(options);
```
Comment (Priya-Patel31) [inline quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: remove this commented code

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            <div className="image-container">
+              <img
+                src={quizImage}
+                className="img-responsive quiz-info-image"
+                alt={quizName}
+              />
+            </div>
+            <div className="quiz-question">
+              <h2 className="font-medium-large weight-semi-bold primary-color">
+                Question {question.questionNumber} / {questions.length}:
```
Comment (Priya-Patel31) [inline quizbizz/src/pages/quizquestion/QuizQuestion.jsx:91]: you can use like this too and why you are not using ? .
For example : question?.questionNumber ?? 0

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+          {/* <!--  ******************************** Quiz Options Container End ********************************  --> */}
+
+          {/* <!--  ******************************** Quiz Button Container Start ********************************  --> */}
+          <div className="quiz-btn-container">
+            <button
+              onClick={handlePrevButtonClick}
+              className="button button-primary quiz-button prev-button"
+              aria-label="previous-button"
+            >
+              <i className="fa fa-angle-left" aria-hidden="true"></i>
```
Comment (Priya-Patel31) [inline quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: You can use react icons

File: quizbizz/src/App.jsx [hunk 5]
```diff
@@ -25,6 +25,7 @@ function App() {
             </RequiredAuth>
           }
         />
+        <Route path="/quizquestion" element={<QuizQuestion />}></Route>
```
Comment (Priya-Patel31) [inline quizbizz/src/App.jsx]: Why you have not wrapped this file with RequiredAuth ??

**Review by Priya-Patel31 — APPROVED** [review]
You can use this for smooth scrolling window.scrollTo({ top: 0, behavior: 'smooth' }) and remove empty files rest your code is very well formatted . You can merge your code .

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+          <div className="quiz-options-container">
+            <h2 className="primary-text-color font-medium-large weight-bold">
+              Choose answer
+            </h2>
+            <div className="spacer-2"></div>
+
+            <ul className="list-style-none" aria-label="options">
+              {options.map((item, index) => (
+                <div>
+                  <li className="quiz-option">
```
Comment (mehulsatardekar) [inline quizbizz/src/pages/quizquestion/QuizQuestion.jsx:114]: make sure to give key for every items

**Review by mehulsatardekar — APPROVED** [review]

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 6 above)
Comment (mehulsatardekar) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx:114]: Rest of the things looks good

**Review by gauravsinhaweb — APPROVED** [review]
code looks good to me, in the UI you can increase the radio button size a bit. Good to merge after the changes mentioned above!

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 2 above)
Comment (Ankur9669) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: Done

File: quizbizz/src/App.jsx (hunk 5 above)
Comment (Ankur9669) [reply quizbizz/src/App.jsx]: Done

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 3 above)
Comment (Ankur9669) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx:91]: Here optional chaining is not required that is why i am not  using it like that

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 4 above)
Comment (Ankur9669) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx]: Done

File: quizbizz/src/pages/quizquestion/QuizQuestion.jsx (hunk 6 above)
Comment (Ankur9669) [reply quizbizz/src/pages/quizquestion/QuizQuestion.jsx:114]: Done

**Pull request merged**